### PR TITLE
Modernize pom

### PR DESCRIPTION
**Pull request by res0nance**
Modernize pom to latest parent POM should help with #35

### CODE REVIEW

**Comment by res0nance** [comment]
Retrigger CI

File: pom.xml [hunk 1]
```diff
@@ -18,16 +18,16 @@
...
     </license>
   </licenses>
 
   <properties>
-    <apacheds.version>2.0.0-M23</apacheds.version>
+    <apacheds.version>2.0.0.AM25</apacheds.version>
     <apacheds.jdbm.version>2.0.0-M3</apacheds.jdbm.version>
-    <jenkins.version>1.651.3</jenkins.version>
-    <jenkins-test-harness.version>2.32</jenkins-test-harness.version>
+    <jenkins.version>2.60.3</jenkins.version>
```
Comment (jvz) [inline pom.xml]: You could use a much more recent version of Jenkins than that. How about:

```suggestion
    <jenkins.version>2.138.4</jenkins.version>
```

Comment (res0nance) [reply pom.xml]: Not sure what the support lifecycle for jenkins. Picked this version just for java 8.
What about how others feel about bumping it again. But I think you have to bump it again for java11

**Comment by rsandell** [comment]
I'll try to remember to merge this directly after I've released 1.21

**Pull request merged**